README.md

```diff
@@ -21,7 +21,14 @@ npm ci
 
 ## Getting Started
 
-To start monitoring URLs, you need to create a configuration file (JSON file). The configuration file contains the probes, alerts, and notification configurations. You can check the sample configuration file in [config.example.json](https://github.com/hyperjumptech/monika/blob/main/config.example.json) or from our [docs](https://github.com/hyperjumptech/monika/quick-start).
+To start monitoring URLs, you need to create a configuration file (JSON file). The configuration file contains the probes, alerts, and notification configurations. You can check the sample configuration file from our [docs](https://github.com/hyperjumptech/monika/quick-start) or copy from one of these files:
+
+- [config.smtp-gmail.example.json](https://github.com/hyperjumptech/monika/blob/main/config.smtp-gmail.example.json)
```

Comment:
can we put those configs inside folder?
such as `config_sample`?

Reply:
agree